streams/src/test/java/org/apache/kafka/streams/integration/RestoreIntegrationTest.java

```diff
@@ -79,14 +91,15 @@
     @BeforeClass
     public static void createTopics() throws InterruptedException {
         CLUSTER.createTopic(INPUT_STREAM, 2, 1);
+        CLUSTER.createTopic(INPUT_STREAM_2, 2, 1);
     }
 
-    private Properties props() {
+    private Properties props(final String applicationId) {
```

Comment:
why do we add this parameter, but not use it in the config? Config is changed to use `this.applicationId` ?

Reply:
it gets used as the application id in the props.

Reply:
After removing the `this`, yes :)

Reply:
nah, you were looking at the wrong line. It was already set as the `APPLICATION_ID_CONFIG`. The `this.applicationId` was just used for creating a Temp directory